Kconfig

```diff
@@ -71,10 +71,17 @@ config CC_COVERAGE
 
 config XEN
 	bool "Select U-Boot be run as a bootloader for XEN Virtual Machine"
-	default n
 	help
 	  Enabling this option will make U-Boot be run as a bootloader
-	  for XEN Virtual Machine.
+	  for XEN [1] Virtual Machine.
+
+	  Xen is a virtual machine monitor (VMM) or a hypervisor with support for
```

Comment:
hmmmmmm. Can we put some text from [this description](https://wiki.xenproject.org/wiki/Xen_Project_Software_Overview)?

Reply:
Sure